core/src/main/java/hudson/node_monitors/ResponseTimeMonitor.java

```diff
@@ -49,6 +48,9 @@
     public static final AbstractNodeMonitorDescriptor<Data> DESCRIPTOR = new AbstractAsyncNodeMonitorDescriptor<Data>() {
         @Override
         protected Callable<Data,IOException> createCallable(Computer c) {
+            if (c.isOffline()) {
```

Comment:
Better to just check that the channel != null and not closing or closed

Otherwise if the agent is taken offline due to the returned metric it cannot come back on-line